source/tutorials/longitudinal.rst

```diff
@@ -95,29 +95,35 @@ Finally, scatter plots categorized by each "group column" are shown at the botto
 
 .. command-block::
 
    qiime longitudinal volatility \
      --m-metadata-file ecam-sample-metadata.tsv \
      --m-metadata-file shannon.qza \
-     --p-metric shannon \
-     --p-group-column delivery \
+     --p-default-metric shannon \
+     --p-default-group-column delivery \
      --p-state-column month \
      --p-individual-id-column studyid \
      --o-visualization volatility.qzv
 
 
-The resulting visualization contains some basic results. First, the "Volatility test parameters" summarizes some key parameters, as well as global mean and standard deviation — these are measured across all samples, regardless of which group they are in.
+In the resulting visualization, a line plot appears on the left-hand side of the plot and a panel of "Plot Controls" appears to the right. These "Plot Controls" interactively adjust several variables and parameters. This allows us to determine how groups' and individuals' values change across a single independent variable, ``state-column``. Interective features in this visualization include:
 
-Second, control charts display the mean value of "metric" at each "state". The first plot shown contains all samples, categorized by group (as defined by ``group-column``); the following plots show each individual group, in order to show their individual control characteristics as described in the rest of this paragraph. The mean for all samples in each plot is shown as a black line. The "control limits", 3 standard deviations above and below the mean, are shown as dashed lines. The "warning limits", 2 standard deviations above and below the mean, are shown as dotted lines. The idea behind this plot is to show how a variable is changing over time (or a gradient) in relation to the mean. Large departures from the mean values can cross the warning/control limits, indicating a major disruption at that state; for example, antibiotic use or other disturbances impacting diversity could be tracked with these plots.
+1. The "Metric column" tab lets us select which continuous metadata values to plot on the y-axis. All continuous numeric columns found in metadata/artifacts input to this action will appear as options in this drop-down tab. In this example, the initial variable plotted in the visualization is shannon diversity because this column was designated by the optional ``default-metric`` parameter.
+2. The "Group column" tab lets us select which categorical metadata values to use for calculating mean values. All categorical numeric columns found in metadata/artifacts input to this action will appear as options in this drop-down tab. These mean values are plotted on the line plot, and the thickness and opacity of these mean lines can be modified using the slider bars in the "Plot Controls" on the right-hand side of the visualization. Error bars (standard deviation) can be toggled on and off with a button in the "Plot Controls".
```

Comment:
"All categorical numeric columns" - should that read "All categorical metadata columns"?